model/godleyTableWindow.cc

```diff
@@ -463,10 +466,12 @@ namespace minsky
         selectedRow=rowY(y);
         if (selectedRow>=0 && selectedRow<int(godleyIcon->table.rows()) &&
             selectedCol>=0 && selectedCol<int(godleyIcon->table.cols()))
-          {
-            selectIdx=insertIdx = textIdx(x);
-            savedText=godleyIcon->table.cell(selectedRow, selectedCol);
-          }
+           {
+             selectIdx=insertIdx = textIdx(x);
+             auto& str=godleyIcon->table.cell(selectedRow,selectedCol);     
+             if (str!=godleyIcon->table.cell(1,0))                               // Cannot save text in cell(1,0). For ticket 1064
```

Comment:
Same - you should just test directly on selectedCol, selectedRow.

Reply:
Done